docs/applications/simple-bank-account/src/main/java/com/scalar/application/bankaccount/service/AccountService.java

```diff
@@ -68,21 +68,22 @@
 
   private ResponseEntity<String> serve(ThrowableFunction f, JsonObject json) {
     try {
-      ContractExecutionResponse response = f.apply(json);
-
-      if (response.getStatus() == StatusCode.OK.get()) {
-        return ResponseEntity.ok(response.getResult());
-      } else {
-        return ResponseEntity.badRequest()
-            .body("status: " + response.getStatus() + ", message: " + response.getMessage());
-      }
+      ContractExecutionResult result = f.apply(json);
+
+      return ResponseEntity
+              .ok(result.getResult().isPresent() ? result.getResult().get().toString() : null);
```

Comment:
```suggestion
              .ok(result.getResult().get());
```

Reply:
Thank you for review comments.
`result.getResult()`  maybe null, so checking is required.

So code modified like this based on discussion
```
return ResponseEntity
              .ok(result.getResult().isPresent() ? result.getResult().get().toString() : "{}");
```